package-lock.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
@veljkocukic there's no need to have this file here since Yarn should be globally installed not locally (in the project).